flink/v1.15/flink/src/main/java/org/apache/iceberg/flink/sink/IcebergStreamWriter.java

```diff
@@ -86,6 +86,10 @@ public void endInput() throws IOException {
     // For bounded stream, it may don't enable the checkpoint mechanism so we'd better to emit the
     // remaining completed files to downstream before closing the writer so that we won't miss any
     // of them.
+
```

Comment:
nit: we can probably remove it here and adjust the comment in the `flush` method

Reply:
I don't think we have changed its nature, so I think we can leave this part of the content